Cargo.toml

```diff
@@ -22,10 +22,8 @@ fluent-syntax = "0.9"
 unic-langid = "0.4.1"
 unicode-segmentation = "1.3.0"
 log = "0.4.8"
+simple_logger = "1.3.0"
```

Comment:
cool, I was thinking we might want to avoid the dependency and just do something simple ourselves, but this doesn't seem to have too much overhead, so I think it's fine. If you wanted to play around with implementing an even simpler logger yourself you'd be welcome, but I'm happy with this.